src/platform/modulesmanager/inc/ModulesManager.h

```diff
@@ -58,8 +58,8 @@ class ModulesManager
     void UnloadModules();
 
 protected:
-    // Component name -> set of reported components according to osconfig.json
-    std::map<std::string, std::set<std::string>> m_reportedComponents;
+    // Component name -> vector of reported components according to osconfig.json

```

Comment:
>-> [](http://example.com/codeflow?start=22&length=2)

What does this "->" really means in the comments in this code? Suggest replacing them all with ':'

Reply:
Changed to remove `->`